src/styles/_catalog.scss

```diff
@@ -13,10 +14,30 @@
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+    box-sizing: border-box;
   }
+}
+
+@media screen and (min-width: 320px) {
+  .catalog {
+    grid-template-columns: 200px;
+  }
+}
+
+@media screen and (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media screen and (min-width: 768px) {
+  .catalog {
```

Comment:
Add 'auto' as the second parameter to 'grid-template-columns' to make it responsive, like this: 'grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));'